.github/scripts/package_lfc.sh

```diff
@@ -16,14 +16,15 @@ mkdir -p $outname/lib
 mv org.lflang/build/libs/org.lflang-*-SNAPSHOT-all.jar $outname/lib
 
 # Move & patch wrappers
-sed -e '/^lfbase=/d' -e 's/\${lfbase}build\/libs/\${base}lib/g' -e '/^if \[\[ ! -f "\$jarpath" \]\]; then/{n;N;N;d}' -e '/^if \[\[ ! -f "\$jarpath" \]\]; then/a \
+sed -e '/^lfbase=/d' -e 's/\${lfbase}build\/libs/\${base}lib/g' -e '/^if \[\[ ! -f "\$jarpath" \]\]; then/{n;N;N;d}' bin/lfc > $outname/bin/lfc
```

Comment:
Why exactly is this patching needed? I am wondering if we should make the lfc script itself a bit more flexible, so that it also works in conjunction with the nightly builds. The patching seems a bit fragile and updating the lfc scripts could break our nighty builds

Reply:
This patching is done so that the scripts are the same as the ones delivered with the alpha release. The reasoning is that you don't want the gradle invocation in there since we ship a finished `jar`. But I agree, it's fragile. Even if you include separate scripts for the nightly things may break if we change the original scripts.

Reply:
Ah, right. That makes sense. Maybe we should just remove gradle from the lfc script altogether? Then the patched version here would be the default. I don't think it is asked too much to run gradle first before running `bin/lfc` the first time. @lhstrh What do you think?

As a replacement of the old functionality, we also have the new `runLfc` gradle task.

Reply:
Regardless what the outcome of this discussion is, I'm gonna merge the PR for now. If a README in the `lfc` zip file or a change to the `bin/lfc` files is required, we can do that separately, I want to see whether the build succeeds tonight. :)